src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -565,12 +565,12 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
     <value>No updates available for this framework packages.</value>
   </data>
   <data name="ListPkg_NoUpdatesForProject" xml:space="preserve">
-    <value>The given project `{0}` have no updates given the current sources.</value>
```

Comment:
It will be helpful if you make this change through VS. 
Next time someone else makes a change here, it will make changes to the designer.cs, and it will also make the extra changes caused by this.

Reply:
Point taken. Feel free to reject this PR and do the same change from VS. I don't have VS setup yet. Will take your help to set it up and run in once E2E so that next time onward the changes are better :)

Reply:
It's a branch in this repo, so I will just update your PR.

Reply:
Running tests, https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=2253163. 

I will merge once the tests are done. 

protip: dev-* branches are run automatically :D